test/generic/FunctionField-test.jl

```diff
@@ -603,6 +603,11 @@ end
       # characteristic 0
       S, y = function_field(P1[i], "y")
 
+      @test divexact(2, y) * y == 2
+      @test divexact(big(2), y) * y == 2
+      @test divexact(1//2, y) * y == 1//2
+      @test divexact(big(1)//2, y) * y == 1//2
```

Comment:
This last new test results in an ambiguity error on master, and is fixed by this PR.